emails/2015intro.md

```diff
@@ -0,0 +1,66 @@
+*This email is a draft. It hasn't been sent to Django Boston yet*
+Hey hey Django Boston!
+
+It's that time again: time for Django Boston to plan its year, and for all of us to start making awesome happen together.
+
+*Quick housekeeping note: John Baldwin and I (Brian Dant) are stepping in for Max Ulenhuth and John Roth. Thanks John and Max for holding it together over the last year and a half!*
+
+Quick refresher on how Django Boston works:
+- Meetups happen on the last Thursday of the month (always at Akamai in Cambridge)
```

Comment:
We've reserved Akamai for the 4th Thursday of every month, not
the last Thursday of every month.